scripts/itkdev-docker-compose-completion.bash

```diff
@@ -9,6 +12,8 @@ _idc_completions()
   local suggestions=($(compgen -W "url open drush mailhog:open mailhog:url sql:connect sql:port xdebug host:insert images:pull composer" -- "${COMP_WORDS[1]}"))
 
   COMPREPLY=("${suggestions[@]}")
+
+  __ltrim_colon_completions "$cur"
```

Comment:
When trying this, I get the error `__ltrim_colon_completions: command not found`. Are there any requirements that must be fulfilled before using this command?

Reply:
It's an new part of bash-completion. Found this online:

> Installing bash-completion (1.2) through brew (brew install bash-completion) solved the issue for me, it might work for you too.